code/drasil-code/lib/Language/Drasil/Code/Imperative/Build/Import.hs

```diff
@@ -29,14 +30,15 @@ data CodeHarness = Ch {
 
 -- | Transforms information in 'CodeHarness' into a list of Makefile rules.
 instance RuleTransformer CodeHarness where
-  makeRule (Ch b r s m d) = maybe [mkRule ["Project Name: " ++ progName m,
-    progPurpAdd m]  buildTarget [] []]
+  makeRule (Ch b r s m d) = maybe [mkRule ["Generated by Drasil " ++ 
+    version drasilMeta,"Project Name: " ++ progName m, progPurpAdd m]  
+    buildTarget [] []]
     (\(BuildConfig comp onm anm bt) ->
     let outnm = maybe (asFragment "") (renderBuildName s m nameOpts) onm
         addnm = maybe (asFragment "") (renderBuildName s m nameOpts) anm
     in [
-    mkRule ["Project Name: " ++ progName m, progPurpAdd m] 
-    buildTarget [outnm] [],
+    mkRule ["Generated by Drasil " ++ version drasilMeta, "Project Name: " ++ 
```

Comment:
The string "Generated by Drasil" appears twice in this file alone...

Reply:
I will change that right now!